apps/web/src/utils/track/usedTrack.ts

```diff
@@ -32,6 +33,10 @@ export default class UsedTrack {
     )
   }
 
+  static checkBuildingPropertyStatus(propertyStatus: BuildingProperty): boolean {
+    return this.findInQuestionnaireData(TrackId.BuildingProperty, QuestionnaireDataEnum.building_property) === propertyStatus
+  }
+
```

Comment:
suggestion: en fait je proposerai bien d'harmoniser cette classe sur le naming de ses fonctions.
Pour moi, toutes les fonctions `isQuelqueChose` devraient être des `hasQuelqueChose`, c'est peut-être un peu de chipotage mais on vérifier si le questionaire contient un/des theme.s, donc `has`.
du coup, je mettrais plutôt un `hasBuildingProperty` avec une vérif sur `propertyStatus !== BuildingProperty.No`.
Je trouve que ça apporte plus de cohérence sur l'usage de cette classe.
T'en penses quoi ?

Reply:
@dolemoine je prends pour les hasTheme au lieu des is mais pour la fonction que j'ai rajouté je me disais que c'était plus efficace d'avoir une fonction de check qui prend en argument un statut plutot que d'avoir tout plein de fonctions

Reply:
praise: Cool ! merci pour la petit refacto. le `is` était un reliquat de quand il y avait encore des objectifs

Par contre, effectivement, pour le `checkBuildingPropertyStatus`, j'aurai plutôt mis un `isBuildingPropertyStatus` aussi ^^